src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/LocalDebugProxyManager.ts

```diff
@@ -0,0 +1,60 @@
+const fetch = require('node-fetch');
+import * as stream from 'stream';
+import * as extract from 'extract-zip';
+import * as net from 'net';
+import * as os from 'os';
+import * as fs from 'fs';
+import * as util from 'util';
+
+export class LocalDebugProxyManager {
```

Comment:
We cannot guarantee that the the user will have the devserver package installed on their machine so we fetch it from NuGet and place it in a temporary location.